daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -111,7 +111,7 @@ private[lf] object Speedy {
       /* Flag to trace usage of get_time builtins */
       var dependsOnTime: Boolean,
       // local contracts, that are contracts created in the current transaction)
-      var localContracts: Map[V.ContractId, (Ref.TypeConName, SValue)],
+      var localContracts: Map[V.ContractId, (Ref.TypeConName, SValue, SValue, SValue)],
```

Comment:
Why not caching the full node ?

Reply:
I was planning to move the `localContracts` Map directly in the `PatialTransaction`. Here all the data are available.

Reply:
you mean cache the create node? No good reason apart from the fact that it gets us back to having to cache the `SValue` computation separately. I mostly just hacked up the simplest thing I could come up with to see if it’s worth pursuing this further or not.